cmd/kola/options.go

```diff
@@ -63,6 +63,7 @@ func init() {
 	sv(&kola.AWSOptions.AMI, "aws-ami", "alpha", `AWS AMI ID, or (alpha|beta|stable) to use the latest image`)
 	sv(&kola.AWSOptions.InstanceType, "aws-type", "t2.small", "AWS instance type")
 	sv(&kola.AWSOptions.SecurityGroup, "aws-sg", "kola", "AWS security group name")
+	sv(&kola.AWSOptions.IAMInstanceProfile, "aws-iam-profile", "kola", "AWS IAM instance profile name")
```

Comment:
Should mention you added this flag in the commit message.

Reply:
For what purpose?

Reply:
Makes it easy to tell when the flag was added. Also, the current commit message implies it is hardcoded and not configurable. Adding a comment that there is a flag to change it clarifies the nature of the change.

Reply:
> Makes it easy to tell when the flag was added.

That's what `git blame` is for.

> Also, the current commit message implies it is hardcoded and not configurable. Adding a comment that there is a flag to change it clarifies the nature of the change.

The flag is pro forma, same as `--aws-sg`.  In practice no one should ever need to use it.

Reply:
> That's what git blame is for

Sure, but if you don't know where these options are (think about new hires, drive by contributers) that doesn't work. 

I'm in general in favor of the commit message including any interface changes, regardless of if we think anyone will use it.